Project.toml

```
@@ -33,6 +34,7 @@ Compat = "2.2, 3"
 Distributions = "0.21, 0.22, 0.23"
 Formatting = "0.4"
 IteratorInterfaceExtensions = "0.1.1, 1"
+MLJModelInterface = "~0.3.5"
```

For pre-1.0 versions, this is actually [the same as the standard caret specifier which can be omitted](https://julialang.github.io/Pkg.jl/v1/compatibility/#Caret-specifiers-1):
```suggestion
MLJModelInterface = "0.3.5"
```

---

OK.